docs/operator-manual/project.yaml

```diff
@@ -3,6 +3,9 @@ kind: AppProject
 metadata:
   name: my-project
   namespace: argocd
+  # Finalizer that ensures that project is not deleted until it is referenced by any application
```

Comment:
Should it be "Finalizer that ensures that project is not deleted until it is _Not_ referenced by any application"

Reply:
nice catch. fixed